target_postgres/target.py

```diff
@@ -12,11 +12,39 @@ class TargetPostgres(Target):
     name = "target-postgres"
     config_jsonschema = th.PropertiesList(
         th.Property(
-            "sqlalchemy_url",
+            "dialect",
             th.StringType,
-            required=True,
-            description="SQLAlchemy connection string, example."
-            + "`postgresql://postgres:postgres@localhost:5432/postgres`",
+            description="The Dialect of SQLAlchamey"
+        ),
+        th.Property(
+            "driver_type",
+            th.StringType,
+            description="The Python Driver you will be using to connect to the SQL server"
+        ),
+        th.Property(
+            "host",
```

Comment:
required=True for this, user, password, and database I think and we'd be good. If they are not set we'd fail with a keyerror from the function above anyway

Reply:
With sqlalchemy url's also being in the list we can't just require these if the other isn't required.